runtime/vm/OutOfLineINL_openj9_internal_foreign_abi_InternalDowncallHandler.cpp

```diff
@@ -141,8 +146,13 @@ OutOfLineINL_openj9_internal_foreign_abi_InternalDowncallHandler_initCifNativeTh
 				goto freeAllMemoryThenExit;
 			/* Only intended for struct as the primitive's ffi_type is non-null */
 			} else if ((NULL == argTypes[argIndex])
-			|| ((FFI_TYPE_STRUCT == argTypes[argIndex]->type) && (NULL == argTypes[argIndex]->elements))
-			) {
+						|| ((FFI_TYPE_STRUCT == argTypes[argIndex]->type
+#if defined(J9ZOS390)
+							|| FFI_TYPE_STRUCT_FF == returnType->type
```

Comment:
Please add brackets for the newly added `if` statements.

Reply:
Changed in https://github.com/eclipse-openj9/openj9/pull/19645/commits/1e752555416853814f4beb0cf0eea5f3fe8580bf